dags/materialized_columns.py

```diff
@@ -141,7 +126,11 @@ def run_materialize_mutations(
     # materialized.
     for partition in sorted(remaining_partitions, reverse=True):
         shard_tasks = {
-            shard_num: MaterializeColumnInPartitionTask(config.table, config.column, partition).run
+            shard_num: MutationRunner(
+                config.table,
+                f"MATERIALIZE COLUMN {config.column} IN PARTITION %(partition)s",
+                {"partition": partition},
+            )
```

Comment:
The `__repr__` of `MutationRunner` will be used in the Dagster logs in place of `MaterializeColumnInPartitionTask.run` - this is partially the motivation for making this change